_data/hosting_providers.json

```diff
@@ -2208,5 +2208,15 @@
     "plan": "",
     "reviewed": "2019.7.22",
     "note": ""
-  }
+  },
+  {
+    "name": "Aquatis",
+    "link": "https://aquatis.host/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
```

Comment:
Is there a page on your site that says that you support TLS by default? We should link to that.

Reply:
On the product page it shows up, didn't know if I should link it or not.
https://you.aquatis.host/cart.php?gid=2

Reply:
hm, you're right, no need. just to check -- there's no steps users need to take or buttons to click to get https support?

Reply:
Yap, they should get https automatically. (we have AutoSSL with cPanel)